catbus-navigator/web-ui
# CN-24: setup prettier

# Description

This PR adds Prettier to our project, to standardize code formatting. 

Fixes # [CN-24](https://jpt4.atlassian.net/browse/CN-24)

# How Has This Been Tested?

- Run `npm i` to install prettier
- Run `npx prettier --check .` in `my-app` to check if files are correctly formatted.
- Run `npx prettier --write .` to format all files automatically.
- Setup VSCode to format files automatically on save, and test this.

# Checklist:

- [x] Have you merged main into this branch and resolved any conflicts?
- [x] Have you updated any necessary documentation?



[CN-24]: https://jpt4.atlassian.net/browse/CN-24?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
